documentation/modules/rn-2.4.adoc

```diff
@@ -0,0 +1,147 @@
+
+When migrating from vSphere to the local OCP cluster, the conversion pod transfers the disk data instead of Containerized Data Importer (CDI), accelerating the migration.
+
+.Migrated virtual machines are not scheduled on the target OCP cluster
+
+The migrated virtual machines are no longer scheduled on the target OCP cluster. This enables migrating VMs that cannot start due to limit constraints at the migration time on the target.
+
+.StorageProfile resource needs to be updated for a non-provisioner storage class
+
+You must update the `StorageProfile` resource with `accessModes` and `volumeMode` for non-provisioner storage classes such as NFS. The documentation includes a link to the relevant procedure.
+
+[id="new-features-and-enhancements-24_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
+
+.OpenStack migration
+
+OpenStack is added as a source provider. Only cold migrations are supported as tech preview in this release.
+
+.OCP console plugin
+
+This version of {project-full} has replaced the stand-alone UI with a UI that is integrated in the OCP Console. The new UI operates as an OCP Console plugin that adds the sub-menu `Migration` to the navigation bar. It is implemented in version 2.4, disabling the old UI. You can enable the old UI by setting `feature_ui: true` in ForkliftController. link:https://issues.redhat.com/browse/MTV-427[(MTV-427)]
+
+.Skip certification option
+
+'Skip certificate validation' option was added to VMware and RHV providers. If checked, the provider's certificate will not be validated and the UI will not ask for specifying a CA certificate.
+
+.Only third-party certificate required
+
+When defining a RHV provider that set with third-party certificate, only the third-party certificate needs to be specified and MTV would retrieve the Manager CA certificate automatically.
+
+.Conversion of VMs with RHEL9 guest operating system
+
+For cold migrations from vSphere to the local OCP cluster, we use virt-v2v on RHEL 9 that supports conversion of virtual machines with RHEL 9 as the guest operating system. link:https://issues.redhat.com/browse/MTV-332[(MTV-332)]
+
+[id="known-issues-24_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+.Deleting migration plan does not remove temporary resources
+
+Deleting a migration plan does not remove temporary resources such as importer pods, conversion pods, config maps, secrets, failed VMs and data volumes. +
+You must archive a migration plan before deleting it in order to clean up the temporary resources. link:https://bugzilla.redhat.com/show_bug.cgi?id=2018974[(BZ#2018974)]
+
+.Unclear error status message for VM with no operating system
+
+The error status message for a VM with no operating system on the *Migration plan details* page of the web console does not describe the reason for the failure. link:https://bugzilla.redhat.com/show_bug.cgi?id=2008846[(BZ#22008846)]
+
```

Comment:
The page is now called Plans -- either change your text or or remove the bold so your text tells the user what they are looking for without specifying its exact title in the UI. Never rely on Jiras for UI details, especially when you can check the UI yourself.